### DISCUSSION TITLE
How safe is "Load missing entries"?

### DISCUSSION THREAD

**Discussion by jan-Sanku** [vrcx-team/VRCX#532] [Q&A] [answered]
There's a warning about it using many API requests, which makes perfect sense. But how many are we talking, and at what point (# of friends to fetch) could it lead to moderation from VRC for spamming the API? Does "use it sparingly" mean no more than once a day, hour, month?

**Comment by Myrkie** [comment]
> There's a warning about it using many API requests, which makes perfect sense. But how many are we talking, and at what point (# of friends to fetch) could it lead to moderation from VRC for spamming the API? Does "use it sparingly" mean no more than once a day, hour, month?

https://github.com/vrcx-team/VRCX/discussions/175#discussioncomment-603416

**Reply by jan-Sanku** [reply]
Oh I should've specified that I've read that post. The reason I'm asking again is because almost 2 years have passed, so surely by now there would be more information on this. Maybe some clarification from the VRC devs, or experiences of VRCX users who did or didn't get banned for using this feature a lot.

**Reply by Myrkie** [reply]
ive used it and never received a ban, its labeled as being unsafe because the amount of requests it can and does make depends on how many friends you have, so the amount of requests varies greatly.

**Reply by jan-Sanku** [reply]
I see. I'm guessing less than 300 friends and very rare usage should be fine then?

**Reply by Myrkie** [reply, marked as answer]
in theory, i have over 500 and have not received any moderation.

**Reply by jan-Sanku** [reply]
good to know, thanks!

**Reply by Natsumi-sama** [reply]
worst case you get locked out of the API for maybe an hour or so. "use it sparingly" means only use it when you need it. Considering the game now does this action every time it starts up I wouldn't worry about it.
